docs/Deployment.md

```diff
@@ -2,24 +2,79 @@
 
 ## Deployment Process
 
+The following should be read as more of a reference than a guide. To deploy
+Rocket 2, you must follow the steps as if you were building it for local use,
+except some tweaks in regards to where it goes and more tooling-specific details
+mentioned below.
+
+### Hosting
+
+Rocket 2 is currently hosted by an AWS EC2 t2.micro instance.
```

Comment:
It might make sense to discuss why this instance in particular was chosen (presumably price?).

Reply:
@cheukyin699 could note another pro is that Inertia has the ability to (kind of) automatically provision an EC2 instance for you, with docs available walking through how to use it: https://inertia.ubclaunchpad.com/#provisioning-a-remote